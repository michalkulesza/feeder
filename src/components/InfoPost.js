import React from "react";

const InfoPost = () => {
  return (
    <div className="post info-post">
      <p>Please log in to see the feed.</p>
    </div>
  );
};

export default InfoPost;
