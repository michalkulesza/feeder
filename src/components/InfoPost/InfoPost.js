import React from "react";

const InfoPost = ({ children }) => {
	return (
		<div className="post info-post">
			<p>{children}</p>
		</div>
	);
};

export default InfoPost;
