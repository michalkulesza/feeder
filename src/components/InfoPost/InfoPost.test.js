import React from "react";
import { shallow } from "enzyme";
import InfoPost from "./InfoPost";

describe("<InfoPost/>", () => {
	it("should match the snapshot", () => {
		const wrapper = shallow(
			<InfoPost>
				<h1>test</h1>
			</InfoPost>
		);

		expect(wrapper.html()).toMatchSnapshot();
	});

	it("should render children props", () => {
		const wrapper = shallow(
			<InfoPost>
				<h1>test</h1>
				<h1>test</h1>
				<h1>test</h1>
			</InfoPost>
		);

		expect(wrapper.find("h1").length).toBe(3);
	});
});
