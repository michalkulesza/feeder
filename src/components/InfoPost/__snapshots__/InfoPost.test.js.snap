// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<InfoPost/> should match the snapshot 1`] = `"<div class=\\"post info-post\\"><p><h1>test</h1></p></div>"`;
